uk_bin_collection/uk_bin_collection/councils/DoverDistrictCouncil.py

```
@@ -1,61 +1,95 @@
-import re
-from datetime import datetime
+from datetime import datetime, timezone
 
-from bs4 import BeautifulSoup
+import requests
 
-from uk_bin_collection.uk_bin_collection.common import *  # Consider specific imports
+from uk_bin_collection.uk_bin_collection.common import *
```

Ruff (0.15.12)

[error] `from uk_bin_collection.uk_bin_collection.common import *` used; unable to detect undefined names

(F403)